packages/@glimmer/runtime/test/partial-test.ts

```diff
@@ -1,470 +1,387 @@
-  rerender(null, { assertStable: true });
-  equalTokens(root, `Before <div>Testing</div> After`);
-});
 
-QUnit.test('static partial with self reference', () => {
-  let template = compile(`{{partial 'birdman'}}`);
+  @test 'static partial with self reference'() {
+    this.env.registerPartial('birdman', `Respeck my {{item}}. When my {{item}} come up put some respeck on it.`);
 
-  env.registerPartial('birdman', `Respeck my {{item}}. When my {{item}} come up put some respeck on it.`);
-  render(template, { item: 'name' });
+    this.render(`{{partial 'birdman'}}`, { item: 'name' });
+    this.assertHTML('Respeck my name. When my name come up put some respeck on it.');
 
-  rerender(null, { assertStable: true });
-
-  equalTokens(root, `Respeck my name. When my name come up put some respeck on it.`);
-  rerender({ item: 'name' }, { assertStable: true });
-  equalTokens(root, `Respeck my name. When my name come up put some respeck on it.`);
-});
-
-QUnit.test('static partial with local reference', () => {
-  let template = compile(`{{#each qualities key='id' as |quality|}}{{partial 'test'}}. {{/each}}`);
-
-  env.registerPartial('test', `You {{quality.value}}`);
-  render(template, { qualities: [{id: 1, value: 'smaht'}, {id: 2, value: 'loyal'}] });
-
-  rerender(null, { assertStable: true });
-
-  equalTokens(root, `You smaht. You loyal. `);
-  rerender({ qualities: [{id: 1, value: 'smaht'}, {id: 2, value: 'loyal'}] }, { assertStable: true });
-  equalTokens(root, `You smaht. You loyal. `);
-});
-
-QUnit.test('static partial with local reference (unknown)', () => {
-  let template = compile(`{{#each qualities key='@index' as |quality|}}{{partial 'test'}}. {{/each}}`);
-
-  env.registerPartial('test', `You {{quality}}`);
-  render(template, { qualities: ['smaht', 'loyal'] });
-
-  rerender(null, { assertStable: true });
-
-  equalTokens(root, `You smaht. You loyal. `);
-  rerender({ qualities: ['smaht', 'loyal'] }, { assertStable: true });
-  equalTokens(root, `You smaht. You loyal. `);
-});
-
-QUnit.test('static partial with named arguments', () => {
-  env.registerBasicComponent('foo-bar', BasicComponent, `<p>{{@foo}}-{{partial 'test'}}</p>`);
+    this.assertStableRerender();
+    this.assertHTML('Respeck my name. When my name come up put some respeck on it.');
+  }
 
-  let template = compile(`<foo-bar @foo={{foo}} @bar={{bar}} />`);
+  @test 'static partial with local reference'() {
+    const props = { qualities: [{id: 1, value: 'smaht'}, {id: 2, value: 'loyal'}] };
+    this.env.registerPartial('test', `You {{quality.value}}`);
 
-  env.registerPartial('test', `{{@foo}}-{{@bar}}`);
-  render(template, { foo: 'foo', bar: 'bar' });
-  equalTokens(root, `<p>foo-foo-bar</p>`);
+    this.render(`{{#each qualities key='id' as |quality|}}{{partial 'test'}}. {{/each}}`, props);
+    this.assertHTML('You smaht. You loyal. ');
+    this.assertStableRerender();
 
-  rerender(null, { assertStable: true });
+    this.rerender(props);
```

Comment:
This is basically doing the same thing as L39. Can you add an update and reset step like I mentioned above.